pkgs/development/compilers/llvm/10/llvm.nix

```diff
@@ -121,6 +122,19 @@ in stdenv.mkDerivation (rec {
   ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
     "-DCMAKE_CROSSCOMPILING=True"
     "-DLLVM_TABLEGEN=${buildPackages.llvm_10}/bin/llvm-tblgen"
+    (
+      let
+        nativeCC = pkgsBuildBuild.stdenv.cc;
+        nativeBintools = nativeCC.bintools.bintools;
```

Comment:
Did you mean to use the unwrapped tools?

Reply:
I needed the tools that provide `ar`, `strip` and `ranlib`. The wrapped tools only provide the tools that are actually wrapped:
```
$ ls $(nix-build --no-out-link -A stdenv.cc.bintools)/bin
as  ld  ld.bfd  ld.gold

$ ls $(nix-build --no-out-link -A stdenv.cc.bintools.bintools)/bin
addr2line  as       dwp      gprof  ld.bfd   nm       objdump  readelf  strings
ar         c++filt  elfedit  ld     ld.gold  objcopy  ranlib   size     strip
```

Reply:
Ah I see. While the wrapped one has a propagated build input, that is no use for the explicit paths below. Sorry!